### DISCUSSION TITLE
Automatically filter out modal particles and some common words that have little meaning for translation.

### DISCUSSION THREAD

**Discussion by despairTK** [Purfview/whisper-standalone-win#138] [Q&A] [answered]
I wonder if this function can be implemented to filter out some modal particles and redundant words that have no impact on translation. You can customize the words that need to be filtered.

Such as these examples:

```
Yes.
Yep.
No.
Okay.
Yeah.
Oops.
Right?
Whoops.
There we go.
```
This type of word can appear if it forms a long sentence with other sentences. Such words will not appear if they occupy a separate line of subtitles.

**Comment by Purfview** [comment, marked as answer]
Look at `--suppress_tokens` option.
